training-intern/src/components/exam3/MultiForm.vue

```diff
@@ -1,15 +1,15 @@
 <template>
-  <div class="exam-3">
+  <div class="exam-1">
     <h3>Exam 3</h3>
   </div>
 </template>
-
-<script>
+  
+  <script>
 export default {};
 </script>
-
-<style>
-.exam-3 {
+  
+  <style>
```

Comment:
toàn bộ style trong component vue nếu không có chủ đích dùng global thì a để trong scoped hết nhé a